collect_app/build.gradle

```diff
@@ -197,9 +197,6 @@ android {
         unitTests {
             includeAndroidResources = true
             returnDefaultValues = true
-            all {
-                maxHeapSize = robolectricHeapSize
```

Comment:
This looks like it might not be relevant and actually be causing problems with newer versions of Robolectric.